utils/encryption_utils.py

```diff
@@ -0,0 +1,17 @@
+LOWERCASE_ASCII_BASE = ord("a")
+UPPERCASE_ASCII_BASE = ord("A")
+ENGLISH_ALPHABET_LENGTH = 26
+
+
+def calculate_shifted_letter(c: str, caesar_cipher_shift: int):
+    if c.islower():
+        return chr(
+            ((ord(c) + caesar_cipher_shift) % LOWERCASE_ASCII_BASE) % ENGLISH_ALPHABET_LENGTH + LOWERCASE_ASCII_BASE)
+    elif c.isupper():
+        return chr(
+            ((ord(c) + caesar_cipher_shift) % UPPERCASE_ASCII_BASE) % ENGLISH_ALPHABET_LENGTH + UPPERCASE_ASCII_BASE)
```

Comment:
This is very not elegant. Why not use `string.ascii_lowercase` and `string.ascii_uppercase` with shifts around the.
I'm sure there is a better way to do this.

also, pretty sure youre first `%` should be a `- ` (minus) sign in lines 9 and 12